github/compliance/report.py

```diff
@@ -311,6 +311,23 @@ class PROCESSING_ACTION(enum.Enum):
     REPORT = 'report'
 
 
+def milestone_for_max_days(
+    max_days: float,
+    delivery_svc_client: delivery.client.DeliveryServiceClient,
+    repository: github3.repos.repo.Repository,
+) -> github3.issues.milestone.Milestone:
+    latest_processing_date = datetime.date.today() + datetime.timedelta(days=max_days)
```

Comment:
pass-in latest_processing_date
name the function to express this semantics (e.g. latest_milestone_before)

Reply:
also: is this really worth factoring out?